### DISCUSSION TITLE
10/17 회화 수업

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#89] [General]
# 10/17 회화 수업

배운 것 I'm weak 보다는 I get sick easily 가 더 자연스럽다. furniture, stuff 는 복수로 쓰지 않는다. 어딘가로 이사를 갈 때는  라고 말하는 것이 좋다. ~를 하기 위해 갔다는 drive up / come up to v-ing 로 표현한다. cloth hanger 는 옷걸이. 행거는 rack 이라고 한다. get ri…

https://vero.wiki/english-speaking-10-17/

**Comment by ukkodeveloper** [comment]
덕분에 영어 마스터했습니다. 지금 당장 미국가요!

**Reply by Cyma-s** [reply]
딜리버리 히어로 지원 ㄱㄱ요
